### PR TITLE
GIX-1715: Use SetDissolveDelay for SNS neurons

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Dragginz SNS project has a max dissolve delay of 888 years.

The manageNeuron endpoint with IncreaseDissolveDelay doesn't support such a large number because it uses nat32.

In this PR: The NNS Dapp uses SetDissolveTimestamp which uses a nat64.

# Changes

* Change sns governance api `increaseDissolveDelay` for `setDissolveDelay` that uses the `SetDissolveTimestamp` manage neuron endpoint.
* Sns neuron service `updateDelay` uses the `setDissolveDelay`.

# Tests

* Substitute sns governance api test in `increaseDissolveDelay` for `setDissolveDelay`.
* Fix broken spec files after refactor.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

File: frontend/src/tests/lib/services/sns-neurons.services.spec.ts [hunk 1]
```diff
@@ -612,51 +612,37 @@ describe("sns-neurons-services", () => {
   });
 
   describe("updateDelay ", () => {
-    const spyOnIncreaseDissolveDelay = jest
-      .spyOn(governanceApi, "increaseDissolveDelay")
+    const spyOnSetDissolveDelay = jest
+      .spyOn(governanceApi, "setDissolveDelay")
       .mockImplementation(() => Promise.resolve());
+    const nowInSeconds = 1689063315;
```
Comment (peterpeterparker) [inline frontend/src/tests/lib/services/sns-neurons.services.spec.ts:618]: is it used elsewhere in test? there are two occurences in this PR, mayby worth a mock for test

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Note: discussed offline, the value to pass is indeed in seconds and not nano seconds.

File: frontend/src/tests/lib/services/sns-neurons.services.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/services/sns-neurons.services.spec.ts:618]: I thought it was easier to have it as a variable here. Other places use other times.

**Pull request merged**